include/zephyr/drivers/flash.h

```diff
@@ -471,6 +471,16 @@ __syscall int flash_ex_op(const struct device *dev, uint16_t code,
 #define FLASH_EX_OP_VENDOR_BASE 0x8000
 #define FLASH_EX_OP_IS_VENDOR(c) ((c) & FLASH_EX_OP_VENDOR_BASE)
 
+/**
+ *  @brief Enumeration for extra flash operations
+ */
+enum flash_ex_op_types {
+	/*
+	 * Sends Flash reset SPI commands to reset the flash device.
```

Comment:
Change description to remove what type of memory is reset. Make it generic reset.